DevelopedCalculator.js

```diff
@@ -0,0 +1,20 @@
+class DevelopedCalculator extends Calculator {
+    history = [];
+    render() {
+        super.render()
+            // first create a div 
+        const historyDiv = document.createElement("div");
+        // add the text to the div
+        historyDiv.appendChild(document.createTextNode("History:" + this.currentResult));
+        // prepend the historyDiv in the host element 
+        this.hostElem.prepend(historyDiv);
+    }
+
+    onButtonNumberClick_(event) {
```

Comment:
why the method name ends with _ ?
any update? 
why you start doing other stuff before finishing working on this?

Reply:
you are asking me for updates

Reply:
yes, because you don't reply on my question, so I have to remind you
now what happen to the method name? why still ends with _ ?
when you are going to answer?